### DISCUSSION TITLE
p/b63ba6ff31c049f8a6703ce7022a79e6/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#23] [Announcements]
# p/b63ba6ff31c049f8a6703ce7022a79e6/

记录一些最常用的命令（从上往下使用频率依次降低），当然，这里只是吾辈个人的，其他功能可能暂时还没用到呢  详细的 git 相关知识强烈推荐 Pro Git  基本命令   关键词 命令格式 命令解释 命令示例    init git init 为当前目录初始化 git 本地仓库    add git add [要追踪的文件/路径] 添加到 git 跟踪 git add -A   commit gi

https://blog.rxliuli.com/p/b63ba6ff31c049f8a6703ce7022a79e6/

**Comment by simonla** [comment]
cat ~/.oh-my-zsh/plugins/git/git.plugin.zsh

有很多快捷方式

**Reply by rxliuli** [reply]
之前都没留意到，更多是通过 git 原始命令或 ide 快捷键之类的
